logger.go

```diff
@@ -0,0 +1,234 @@
+/*
+ Copyright 2016 Derrick J. Wippler. All rights reserved.
+ Use of this source code is governed by a BSD-style
+ license that can be found in the LICENSE file.
+
+COPY NOTICE
+wrapWriter() with associated code and struct's copied from CHI middleware
+ at https://github.com/pressly/chi/blob/master/middleware/logger.go with modifications
+*/
+
+package detka
+
+import (
+	"bufio"
+	"io"
+	"net"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/Sirupsen/logrus"
```

Comment:
I like the way you think :) ! We started using structured logging some time ago. But I don't think we do it enough.